### ISSUE TITLE
Cannot use context menu with keyboard

### ISSUE THREAD

**Issue by barmac** [bpmn-io/dmn-js#848]
### Describe the Bug

The context menu does not take the focus and I cannot take any action in it with keyboard.

![image](https://github.com/bpmn-io/dmn-js/assets/28307541/2d2324c5-4be5-48bb-b9ba-b85b4cc606ac)

How to open context menu with keyboard on Mac (thanks @lmbateman): https://support.apple.com/en-gb/guide/mac-help/mh27469/mac

### Steps to Reproduce

1. Open context menu.
2. Try to move around with arrow keys.

### Expected Behavior

Context menu should work exactly like a system context menu:

* [ ] Be a focus trap
* [ ] Change selection with arrow keys
* [ ] Select and perform action with enter

It closes with escape or outside click already, which otherwise would have been another requirement.

### Environment

Please complete the following information:

 - Browser: [e.g. IE 11, Chrome 69]
 - OS: [e.g. Windows 7]
 - Library version: [e.g. 2.0.0]

**Comment by barmac** [comment]
Note that this issue is not detected by axe.

**Comment by barmac** [comment]
I am not working on this right now. As this is not essential for axe-compliance, I am moving this to the backlog.
